### ISSUE TITLE
Install issue

### ISSUE THREAD

**Issue by JiDai** [GEOSwift/GEOSwift#65]
Hi,

I have a fresh install of Xcode. I change dependencies in `Podfile`
I got the `autoconf not found` error then I install `brew install autoconf automake libtool`
After relâche `pod install`, I got the following error (I tried `pod update` too): 

```[!] /usr/local/bin/bash -c
set -e
type -P autoconf &>/dev/null || alias autoconf 'xcrun autoconf'
type -P autoheader &>/dev/null || alias autoheader 'xcrun autoheader'
type -P aclocal &>/dev/null || alias aclocal 'xcrun aclocal'
type -P automake &>/dev/null || alias automake 'xcrun automake'
type -P glibtool &>/dev/null || alias glibtool 'xcrun glibtool'
type -P glibtoolize &>/dev/null || alias glibtoolize 'xcrun glibtoolize'

sh autogen.sh
./configure
./tools/svn_repo_revision.sh

sed -i "" "s/\/\* #undef HAVE_INT64_T_64 \*\//#define HAVE_INT64_T_64 1/" include/geos/platform.h
sed -i "" "s/#define HAVE_LONG_INT_64 1/\/\* #undef HAVE_LONG_INT_64 \*\//" include/geos/platform.h

cat <<EOT >> include/geos/platform.h
  #undef ISNAN
  #define ISNAN(x) (std::isnan(x))
EOT

patch capi/geos_c.h <<EOF
  149c149
  < #include <geos/export.h>
  ---
  > #define GEOS_DLL
EOF

* Running /usr/local/bin/glibtoolize (2.4.6)
   OPTIONS = --force --copy
glibtoolize: putting auxiliary files in '.'.
glibtoolize: copying file './ltmain.sh'
glibtoolize: putting macros in AC_CONFIG_MACRO_DIRS, 'macros'.
glibtoolize: copying file 'macros/libtool.m4'
glibtoolize: copying file 'macros/ltoptions.m4'
glibtoolize: copying file 'macros/ltsugar.m4'
glibtoolize: copying file 'macros/ltversion.m4'
glibtoolize: copying file 'macros/lt~obsolete.m4'
* Running /usr/local/bin/aclocal (1.15.1)
* Running /usr/local/bin/autoheader (2.69)
* Running /usr/local/bin/automake (1.15.1)
   OPTIONS = --add-missing --copy -Woverride
* Running /usr/local/bin/autoconf (2.69)
======================================
Now you are ready to run './configure'
======================================
checking build system type... x86_64-apple-darwin17.2.0
checking host system type... x86_64-apple-darwin17.2.0
checking target system type... x86_64-apple-darwin17.2.0
checking for a BSD-compatible install... /usr/local/bin/ginstall -c
checking whether build environment is sane... yes
checking for a thread-safe mkdir -p... /usr/local/bin/gmkdir -p
checking for gawk... no
checking for mawk... no
checking for nawk... no
checking for awk... awk
checking whether make sets $(MAKE)... yes
checking whether make supports nested variables... yes
checking whether to enable maintainer-specific portions of Makefiles... no
checking for gcc... gcc
checking whether the C compiler works... yes
checking for C compiler default output file name... a.out
checking for suffix of executables...
checking whether we are cross compiling... no
checking for suffix of object files... o
checking whether we are using the GNU C compiler... yes
checking whether gcc accepts -g... yes
checking for gcc option to accept ISO C89... none needed
checking whether gcc understands -c and -o together... yes
checking for style of include used by make... GNU
checking dependency style of gcc... gcc3
checking how to print strings... printf
checking for a sed that does not truncate output... /usr/local/bin/sed
checking for grep that handles long lines and -e... /usr/bin/grep
checking for egrep... /usr/bin/grep -E
checking for fgrep... /usr/bin/grep -F
checking for ld used by gcc... /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ld
checking if the linker (/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ld) is GNU ld... no
checking for BSD- or MS-compatible name lister (nm)... /usr/bin/nm -B
checking the name lister (/usr/bin/nm -B) interface... BSD nm
checking whether ln -s works... yes
checking the maximum length of command line arguments... 196608
checking how to convert x86_64-apple-darwin17.2.0 file names to x86_64-apple-darwin17.2.0 format... func_convert_file_noop
checking how to convert x86_64-apple-darwin17.2.0 file names to toolchain format... func_convert_file_noop
checking for /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ld option to reload object files... -r
checking for objdump... objdump
checking how to recognize dependent libraries... pass_all
checking for dlltool... dlltool
checking how to associate runtime and link libraries... printf %s\n
checking for ar... ar
checking for archiver @FILE support... no
checking for strip... strip
checking for ranlib... ranlib
checking command to parse /usr/bin/nm -B output from gcc object... ok
checking for sysroot... no
checking for a working dd... /bin/dd
checking how to truncate binary pipes... /bin/dd bs=4096 count=1
checking for mt... no
checking if : is a manifest tool... no
checking for dsymutil... dsymutil
checking for nmedit... nmedit
checking for lipo... lipo
checking for otool... otool
checking for otool64... no
checking for -single_module linker flag... yes
checking for -exported_symbols_list linker flag... yes
checking for -force_load linker flag... yes
checking how to run the C preprocessor... gcc -E
checking for ANSI C header files... yes
checking for sys/types.h... yes
checking for sys/stat.h... yes
checking for stdlib.h... yes
checking for string.h... yes
checking for memory.h... yes
checking for strings.h... yes
checking for inttypes.h... yes
checking for stdint.h... yes
checking for unistd.h... yes
checking for dlfcn.h... yes
checking for objdir... .libs
checking if gcc supports -fno-rtti -fno-exceptions... yes
checking for gcc option to produce PIC... -fno-common -DPIC
checking if gcc PIC flag -fno-common -DPIC works... yes
checking if gcc static flag -static works... no
checking if gcc supports -c -o file.o... yes
checking if gcc supports -c -o file.o... (cached) yes
checking whether the gcc linker (/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ld) supports shared libraries... yes
checking dynamic linker characteristics... darwin17.2.0 dyld
checking how to hardcode library paths into programs... immediate
checking for dlopen in -ldl... yes
checking whether a program can dlopen itself... yes
checking whether a statically linked program can dlopen itself... yes
checking whether stripping libraries is possible... yes
checking if libtool supports shared libraries... yes
checking whether to build shared libraries... yes
checking whether to build static libraries... yes
checking for g++... g++
checking whether we are using the GNU C++ compiler... yes
checking whether g++ accepts -g... yes
checking dependency style of g++... gcc3
checking how to run the C++ preprocessor... g++ -E
checking for ld used by g++... /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ld
checking if the linker (/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ld) is GNU ld... no
checking whether the g++ linker (/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ld) supports shared libraries... yes
checking for g++ option to produce PIC... -fno-common -DPIC
checking if g++ PIC flag -fno-common -DPIC works... yes
checking if g++ static flag -static works... no
checking if g++ supports -c -o file.o... yes
checking if g++ supports -c -o file.o... (cached) yes
checking whether the g++ linker (/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ld) supports shared libraries... yes
checking dynamic linker characteristics... darwin17.2.0 dyld
checking how to hardcode library paths into programs... immediate
checking for library containing strerror... none required
checking whether make sets $(MAKE)... (cached) yes
checking for dirent.h that defines DIR... yes
checking for library containing opendir... none required
checking whether closedir returns void... no
checking for working memcmp... yes
checking for strftime... yes
checking for vprintf... yes
checking for _doprnt... no
checking for size_t... yes
checking for working alloca.h... yes
checking for alloca... yes
checking for dirent.h that defines DIR... (cached) yes
checking for library containing opendir... (cached) none required
checking for ANSI C header files... (cached) yes
checking for memory.h... (cached) yes
checking for unistd.h... (cached) yes
checking ieeefp.h usability... no
checking ieeefp.h presence... no
checking for ieeefp.h... no
checking sys/file.h usability... yes
checking sys/file.h presence... yes
checking for sys/file.h... yes
checking sys/time.h usability... yes
checking sys/time.h presence... yes
checking for sys/time.h... yes
checking for strchr... yes
checking for memcpy... yes
checking for gettimeofday... yes
checking whether stat file-mode macros are broken... no
checking whether struct tm is in sys/time.h or time.h... time.h
checking for size_t... (cached) yes
checking for an ANSI C-conforming const... yes
checking if requested to force inline functions... yes
checking if requested to enable assert macros... yes
checking if requested libstdc++ debug mode... no
checking if g++ supports -pedantic... yes
checking if g++ supports -Wall... yes
checking if g++ supports -ansi... yes
checking if g++ supports -Wno-long-long... yes
checking if g++ supports -ffloat-store... no
checking for finite... yes
checking for isfinite... yes
checking for isnan... yes
checking whether int64_t is 64 bits... no
checking whether long long int is 64 bits... yes
checking OS-specific settings... darwin17.2.0
checking for OS/X version... Mac OS X (Darwin 17.2.0 kernel)
checking that generated files are newer than configure... done
configure: creating ./config.status
config.status: creating Makefile
config.status: creating capi/Makefile
config.status: creating capi/geos_c.h
config.status: creating doc/Doxyfile
config.status: creating doc/Makefile
config.status: creating macros/Makefile
config.status: creating src/Makefile
config.status: creating src/algorithm/Makefile
config.status: creating src/algorithm/locate/Makefile
config.status: creating src/algorithm/distance/Makefile
config.status: creating src/geom/Makefile
config.status: creating src/geom/prep/Makefile
config.status: creating src/geom/util/Makefile
config.status: creating src/geomgraph/Makefile
config.status: creating src/geomgraph/index/Makefile
config.status: creating include/Makefile
config.status: creating include/geos/Makefile
config.status: creating include/geos/algorithm/Makefile
config.status: creating include/geos/algorithm/locate/Makefile
config.status: creating include/geos/algorithm/distance/Makefile
config.status: creating include/geos/geom/Makefile
config.status: creating include/geos/geom/prep/Makefile
config.status: creating include/geos/geom/util/Makefile
config.status: creating include/geos/geomgraph/Makefile
config.status: creating include/geos/geomgraph/index/Makefile
config.status: creating include/geos/index/Makefile
config.status: creating include/geos/index/bintree/Makefile
config.status: creating include/geos/index/chain/Makefile
config.status: creating include/geos/index/intervalrtree/Makefile
config.status: creating include/geos/index/quadtree/Makefile
config.status: creating include/geos/index/strtree/Makefile
config.status: creating include/geos/index/sweepline/Makefile
config.status: creating include/geos/io/Makefile
config.status: creating include/geos/linearref/Makefile
config.status: creating include/geos/noding/Makefile
config.status: creating include/geos/noding/snapround/Makefile
config.status: creating include/geos/operation/Makefile
config.status: creating include/geos/operation/buffer/Makefile
config.status: creating include/geos/operation/distance/Makefile
config.status: creating include/geos/operation/intersection/Makefile
config.status: creating include/geos/operation/linemerge/Makefile
config.status: creating include/geos/operation/overlay/Makefile
config.status: creating include/geos/operation/overlay/snap/Makefile
config.status: creating include/geos/operation/polygonize/Makefile
config.status: creating include/geos/operation/predicate/Makefile
config.status: creating include/geos/operation/relate/Makefile
config.status: creating include/geos/operation/sharedpaths/Makefile
config.status: creating include/geos/operation/union/Makefile
config.status: creating include/geos/operation/valid/Makefile
config.status: creating include/geos/planargraph/Makefile
config.status: creating include/geos/planargraph/algorithm/Makefile
config.status: creating include/geos/precision/Makefile
config.status: creating include/geos/simplify/Makefile
config.status: creating include/geos/triangulate/Makefile
config.status: creating include/geos/triangulate/quadedge/Makefile
config.status: creating include/geos/util/Makefile
config.status: creating include/geos/version.h
config.status: creating src/index/Makefile
config.status: creating src/index/bintree/Makefile
config.status: creating src/index/chain/Makefile
config.status: creating src/index/intervalrtree/Makefile
config.status: creating src/index/quadtree/Makefile
config.status: creating src/index/strtree/Makefile
config.status: creating src/index/sweepline/Makefile
config.status: creating src/io/Makefile
config.status: creating src/linearref/Makefile
config.status: creating src/noding/Makefile
config.status: creating src/noding/snapround/Makefile
config.status: creating src/operation/Makefile
config.status: creating src/operation/buffer/Makefile
config.status: creating src/operation/distance/Makefile
config.status: creating src/operation/intersection/Makefile
config.status: creating src/operation/linemerge/Makefile
config.status: creating src/operation/overlay/Makefile
config.status: creating src/operation/polygonize/Makefile
config.status: creating src/operation/predicate/Makefile
config.status: creating src/operation/relate/Makefile
config.status: creating src/operation/sharedpaths/Makefile
config.status: creating src/operation/union/Makefile
config.status: creating src/operation/valid/Makefile
config.status: creating src/planargraph/Makefile
config.status: creating src/precision/Makefile
config.status: creating src/simplify/Makefile
config.status: creating src/triangulate/Makefile
config.status: creating src/triangulate/quadedge/Makefile
config.status: creating src/util/Makefile
config.status: creating swig/geos.i
config.status: creating swig/Makefile
config.status: creating swig/python/Makefile
config.status: creating swig/python/tests/Makefile
config.status: creating swig/ruby/Makefile
config.status: creating swig/ruby/test/Makefile
config.status: creating php/Makefile
config.status: creating php/test/Makefile
config.status: creating tests/Makefile
config.status: creating tests/bigtest/Makefile
config.status: creating tests/unit/Makefile
config.status: creating tests/perf/Makefile
config.status: creating tests/perf/operation/Makefile
config.status: creating tests/perf/operation/buffer/Makefile
config.status: creating tests/perf/operation/predicate/Makefile
config.status: creating tests/perf/capi/Makefile
config.status: creating tests/xmltester/Makefile
config.status: creating tests/geostest/Makefile
config.status: creating tests/thread/Makefile
config.status: creating tools/Makefile
config.status: creating tools/geos-config
config.status: creating include/config.h
config.status: creating include/geos/platform.h
config.status: executing depfiles commands
config.status: executing libtool commands
Swig: false
Python bindings: false
Ruby bindings: false
PHP bindings: false
#define GEOS_SVN_REVISION 0
configure.ac:37: warning: AM_INIT_AUTOMAKE: two- and three-arguments forms are deprecated.  For more info, see:
configure.ac:37: http://www.gnu.org/software/automake/manual/automake.html#Modernize-AM_005fINIT_005fAUTOMAKE-invocation
configure.ac:40: installing './compile'
configure.ac:9: installing './config.guess'
configure.ac:9: installing './config.sub'
configure.ac:37: installing './install-sh'
configure.ac:37: installing './missing'
capi/Makefile.am: installing './depcomp'
parallel-tests: installing './test-driver'
swig/python/Makefile.am:18: installing './py-compile'
Can't fetch local revision (neither .svn nor .git found)
Wrote rev '0' in file './geos_svn_revision.h'
sed: can't read s/\/\* #undef HAVE_INT64_T_64 \*\//#define HAVE_INT64_T_64 1/: No such file or directory```

**Comment by vfn** [comment]
Can you confirm what Xcode version you are using, as well as cocoapods?

**Comment by JiDai** [comment]
Xcode Version 9.1 (9B55)

```
$ pod --version
1.3.1
```

**Comment by vfn** [comment]
@JiDai thanks! I have the exact same setup, although pod install works as expected on a fresh clone of the repo.

So, you cloned the repo and run `pod install` and get the error above, is that correct? Or are you integrating `GEOSwift` into your project?

If it's an integration issue, can you plz share your `Podfile`

**Comment by JiDai** [comment]
I did not clone the repo, I create new empty project in Swift. modify the podfile and install.

Here is my `Podfile`

```
platform :ios, '8.0'
use_frameworks!

target 'PastSalesExplorer' do
    pod 'GEOSwift'
end
```

**Comment by vfn** [comment]
@JiDai are you still experiencing this problem? If yes, could you please create a project that reproduces the issue and push it into a public github repo and share the link here?

Thanks!

**Comment by JiDai** [comment]
I juste created new project (Single View App), add podfile and install. Here is he repo https://github.com/GEOSwift/GEOSwift

**Comment by vfn** [comment]
You're getting the same issue on this new project?

I cloned your repo and couldn't replicate it:

```
pod install
Analyzing dependencies
Downloading dependencies
Installing GEOSwift (2.1.0)
Installing geos (3.5.0)
Generating Pods project
Integrating client project

[!] Please close any current Xcode sessions and use `DemoGeoSwift.xcworkspace` for this project from now on.
Sending stats
Pod installation complete! There is 1 dependency from the Podfile and 2 total pods installed.
```

And the project running:
<img width="1394" alt="screen shot 2017-12-21 at 10 12 31 am" src="https://user-images.githubusercontent.com/308592/34233121-a31f1dbe-e637-11e7-90c8-ee6494efdf36.png">

**Comment by JiDai** [comment]
Damn. 
Yes, same issue on this new project.
So it is my env ?

**Comment by JiDai** [comment]
I found similar issues on Geos trac project. Can you bump dependency ?

**Comment by vfn** [comment]
@JiDai What do you mean by `Can you bump dependency ?`

**Comment by JiDai** [comment]
Geos has been updated to 3.6.2. I supposed you could upgrade geos to this version but your pod file does not specify version, so the version installed is 3.5. Geos has not publish a version ulterior to 3.5 on cocoa pods ?

**Comment by vfn** [comment]
@JiDai would you like to have a go at that? The GEOS podspec repo is at https://github.com/GEOSwift/geos

Please update it and then create a pull request for the new GEOS pod and another PR on this repo if needed.

Thanks!

**Comment by adampitchie** [comment]
I had the same issue. However, after looking at the CLI output, I found that I just needed to install `autoconf` and `automake`. These can easily be done with homebrew.

Once I did that, I was able to `pod install` without any errors being thrown.

Hope this helps!

**Comment by davenquinn** [comment]
Hi, I don't know if anything has been resolved with this, but I am getting a similar error on attempted install on High Sierra. Essentially the same output as above, ending with
```
Can't fetch local revision (neither .svn nor .git found)
Wrote rev '0' in file './geos_svn_revision.h'
sed: can't read s/\/\* #undef HAVE_INT64_T_64 \*\//#define HAVE_INT64_T_64 1/: No such file or directory
```
([full logs](https://github.com/GEOSwift/GEOSwift/files/1988844/geoswift-install-log.txt) attached.)

This occurs whether I try to integrate with my project, or do a clean install by cloning this repo and then running `pod update` and `pod install`. My `cocoapods` version is 1.5.0. It appears it is trying to install geos 3.5.0.

Any ideas?

**Comment by vfn** [comment]
@davenquinn Can you confirm what version of the library you are trying to install? I released v2.2.0 last week. If you don't have the latest version, can you please try that out?

**Comment by macdrevx** [comment]
We've revised the pod install process for GEOSwift 3.0.0 and geos 3.7.0. Please try them out and if you have any problems, reopen this issue with new info.
